tests/Group.test.ts

```diff
@@ -290,60 +290,78 @@ describe('Group names', () => {
             groupBy: 'path',
             taskLine: '- [ ] a',
             path: 'a/b/c.md',
-            expectedGroupName: 'a/b/c',
+            expectedGroupNames: ['a/b/c'],
         },
 
         // -----------------------------------------------------------
         // group by scheduled
         {
             groupBy: 'scheduled',
             taskLine: '- [ ] a ⏳ 1970-01-01',
-            expectedGroupName: '1970-01-01 Thursday',
+            expectedGroupNames: ['1970-01-01 Thursday'],
         },
         {
             groupBy: 'scheduled',
             taskLine: '- [ ] a',
-            expectedGroupName: 'No scheduled date',
+            expectedGroupNames: ['No scheduled date'],
         },
 
         // -----------------------------------------------------------
         // group by start
         {
             groupBy: 'start',
             taskLine: '- [ ] a 🛫 1970-01-01',
-            expectedGroupName: '1970-01-01 Thursday',
+            expectedGroupNames: ['1970-01-01 Thursday'],
         },
         {
             groupBy: 'start',
             taskLine: '- [ ] a',
-            expectedGroupName: 'No start date',
+            expectedGroupNames: ['No start date'],
         },
 
         // -----------------------------------------------------------
         // group by status
         {
             groupBy: 'status',
             taskLine: '- [ ] a',
-            expectedGroupName: 'Todo',
+            expectedGroupNames: ['Todo'],
         },
         {
             groupBy: 'status',
             taskLine: '- [x] a',
-            expectedGroupName: 'Done',
+            expectedGroupNames: ['Done'],
+        },
+
+        // -----------------------------------------------------------
+        // group by tags
+        {
+            groupBy: 'tags',
+            taskLine: '- [ ] a #tag1',
+            expectedGroupNames: ['#tag1'],
+        },
+        {
+            groupBy: 'tags',
+            taskLine: '- [ ] a #tag1 #tag2',
+            expectedGroupNames: ['#tag1', '#tag2'],
+        },
+        {
+            groupBy: 'tags',
+            taskLine: '- [x] a',
+            expectedGroupNames: ['(No tags)'],
```

Comment:
I'm not sure where to add it, but I feel like these new tests are fine for the tags feature, but there is a whole load of new capability in the storage of groups now, like the ability to have tasks in more than one group - which also opens up the possibility of needing to test the method that returns the number of tasks.

Reply:
There is a test that covers this now: 'should create nested headings if multiple groups used' - thank you.